Assets/Scripts/Game/UserInterfaceWindows/DaggerfallGuildServiceDonation.cs

```diff
@@ -65,7 +65,7 @@ protected virtual void DonationMsgBox_OnGotUserInput(DaggerfallInputMessageBox s
 
                     // Change reputation
                     int rep = Math.Abs(playerEntity.FactionData.GetReputation(factionId));
-                    if (Dice100.SuccessRoll(2 * amount / rep + 1))
+                    if (Dice100.SuccessRoll(2 * amount / ((2 * amount / Math.Max(rep, 1)) + 1)))
```

Comment:
I think you messed up the copy-paste, now the `2 * amount /` part is there twice

Reply:
Where is my brain today, thank you!